mlir/test/Dialect/MIOpen/lowering_filter_tensor_ckyx_cnhw_knhw.mlir

```diff
@@ -46,9 +46,9 @@ func @miopen_conv2d_bwd_data_ckyx_cnhw_knhw(%filter : memref<1x8x128x3x3xf32>, %
 // CHECK:       names = ["gemmG"]
 // CHECK:       source_names = ["g"]
 // CHECK:       names = ["gemmK"]
-// CHECK:       source_names = ["k"]
+// CHECK:       source_names = ["k", "ydotslice", "xdotslice"]
 // CHECK:       names = ["gemmM"]
-// CHECK:       source_names = ["c", "y", "x"]
+// CHECK:       source_names = ["c", "ytildaslice", "xtildaslice"]
```

Comment:
Number of GEMMs: YTilda * XTilda
GemmM = C
GemmN = N * HTildaSlice * WTildaSlice
GemmK = K * YDotSlice * XDotSlice

Number of GEMM is missing
GemmM is wrong

Reply:
There  is not transformation  "Freeze", I think  GemmM could merge C , "ytildaslice" and "xtildaslice".
This unit test is just  test filter tensor, so not GemmN